src/content/en/2019/media.md

```diff
@@ -8,316 +7,276 @@ authors: [colinbendell, dougsillars]
 discuss: 1759
 reviewers: [ahmadawais, eeeps]
 published: 2019-11-11T00:00:00.000Z
-last_updated: 2019-11-15T00:00:00.000Z
+last_updated: 2019-11-16T00:00:00.000Z
 ---
 
 ## Introduction
 Images, animations, and videos are an important part of the web experience. They are important for many reasons - they help tell stories, engage audiences and provide artistic expression in ways that often cannot be easily produced with other web technologies. The importance of these media resources can be demonstrated in two ways: by the sheer volume of bytes required to download for a page, and also the volume of pixels painted with media. 
 
-From a pure bytes perspective, HTTP Archive has historically reported an average of two-thirds of resource bytes associated from media. From a distribution perspective, we can see that virtually every web page depends on images and videos. Even at the 10th percentile, we see that 43% of the bytes are from media and can rise to 91.3% of the total bytes at the 90th percentile of pages. 
+From a pure bytes perspective, HTTP Archive has historically reported an average of two-thirds of resource bytes associated from media. From a distribution perspective, we can see that virtually every web page depends on images and videos. Even at the tenth percentile, we see that 43% of the bytes are from media and can rise to 91.3% of the total bytes at the 90th percentile of pages. 
 
 <figure>
-  <iframe aria-labelledby="fig1-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1189524305&format=interactive"></iframe>
-  <a href="/static/images/2019/04_Media/fig1.png" class="fig-mobile">
-    <img src="/static/images/2019/04_Media/fig1.png" aria-labelledby="fig1-caption" width="600">
+  <iframe title="Figure 1. Web page bytes: image and video versus other." aria-labelledby="fig1-caption" aria-describedby="fig1-description" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1189524305&format=interactive"></iframe>
+  <a href="/static/images/2019/04_Media/fig1_bytes_images_and_video_versus_other.png" class="fig-mobile">
+    <img src="/static/images/2019/04_Media/fig1_bytes_images_and_video_versus_other.png" alt="" alt="Figure 1. Web page bytes: image and video versus other." aria-labelledby="fig1-caption" aria-describedby="fig1-description" width="600">
```

Comment:
Ditto, we should only need one labelled/describedby.

There are also two alt attributes on this.

If we really need all of these redundant attributes it might be best to add them in the chapter generation script. That way we only write things once.

Reply:
Good spot - fixed.

See other comment in regards to the "duplication" and the fact that at least some of them are auto generated.